include/fastdds_monitor/utils.h

```diff
@@ -58,6 +58,10 @@ std::string to_string(
 std::string now(
         bool miliseconds = true);
 
+//! Cast a double to string removing the trailing zeros
+std::string double_to_string(
```

Comment:
As it is going to be used from here as well, erase it from `io` directory.

Reply:
Addressed in e6e4164